src/visual.ts

```diff
@@ -202,6 +203,7 @@ export class DualKpi implements IVisual {
     private target: HTMLElement;
     private size: DualKpiSize;
     private sizeCssClass: DualKpiSizeClass;
+    private events:IVisualEventService
```

Comment:
1. Let's rename it to `eventService`, it will better represent its purpose. 
2. Keep correct spacing `events: IVisualEventService`